src/game_system.cpp

```diff
@@ -574,3 +574,7 @@ void Game_System::IncFrameCounter() {
 Color Game_System::GetBackgroundColor() {
 	return bg_color;
 }
+
+const bool& Game_System::GetInvertAnimations() {
```

Comment:
Primitive types like ints and bools should be returned by value, not `const ref`. References are less efficient for anything smaller than 2 words.